### PR TITLE
chore: Update style guide

### PR DESCRIPTION
**Pull request by AlexXuan233**
## Summary

Update style guide

## Related Issues

None

## Review Hints

- Update project style guide since we have an organization Style Guide

### CODE REVIEW

File: docs/STYLE_GUIDE.md [hunk 1]
```diff
@@ -2,31 +2,18 @@
...
 
-## General Guidelines
+## ComplyTime Organization Style Guide
 
-- **File Naming**: Use lowercase letters and underscores for file names (e.g., `my_file.go`).
-- **Package Names**: Use short, concise, and lowercase names for packages. Avoid underscores and mixed caps.
-- **Error Handling**: Always check for errors and handle them appropriately. Return errors to the caller when necessary.
-- **Testing**: Write tests for your code. Use descriptive names for test functions and include edge cases.
-- **Line Length**: Limit lines to 99 characters when reasonable to improve readability.
+Refer to [Style Guide](https://github.com/complytime/community/blob/main/STYLE_GUIDE.md), this is the universal style guide all projects under ComplyTime Organization should be followed.
```
Comment (sonupreetam) [inline docs/STYLE_GUIDE.md]: @AlexXuan233 Minor edit, "this is the universal style guide that all projects under the ComplyTime Organization should follow".

Comment (AlexXuan233) [reply docs/STYLE_GUIDE.md]: Updated. Thank you.

**Review by marcusburghardt — APPROVED** [review]
LGTM

**Comment by AlexXuan233** [comment]
@sonupreetam Hi, Sonu. Can you help take a look again.

**Review by sonupreetam — APPROVED** [review]
LGTM
